### ISSUE TITLE
2.2.3 Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1413]
- #1417
- #1422

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick c588c0e98cb922ed425626c7e2f27737bfd693fb
git cherry-pick bd1a7b8adc69254f3a5cc990612ab8a0e0f4b7df
```

**Comment by joeparsons** [comment]
## Release Notes Snippet

- Closes #1421: Update Drupal core to 9.2.16 for SA-CORE-2022-006. (#1422)
- Fixes #1415: Add required default_theme property to all tests. (#1417)

**Comment by joeparsons** [comment]
Released.
